### PR TITLE
Fix tech debt

### PR DESCRIPTION
**Pull request by MickStein**
https://eaflood.atlassian.net/browse/LTFRI-1233

Deal with remaining tech debt issues.

### CODE REVIEW

File: node/risk-app/server/plugins/views.js [hunk 1]
```diff
@@ -44,14 +44,14 @@ module.exports = {
...
       phase: 'beta',
       feedback: true,
-      siteUrl: config.floodRiskUrl,
+      siteUrl: config.floodRiskUrl ? config.floodRiskUrl : process.env.FLOOD_RISK_URL,
       ogDescription: 'Check your risk of flooding and use flood risk maps',
       noIndex: true, // stop robot crawl by default
-      friendlyCaptchaEnabled: config.friendlyCaptchaEnabled,
-      friendlyCaptchaSiteKey: config.friendlyCaptchaSiteKey
+      friendlyCaptchaEnabled: config.friendlyCaptchaEnabled ? config.friendlyCaptchaEnabled : process.env.FRIENDLY_CAPTCHA_ENABLED,
+      friendlyCaptchaSiteKey: config.friendlyCaptchaSiteKey ? config.friendlyCaptchaSiteKey : process.env.FRIENDLY_CAPTCHA_SITE_KEY
```
Comment (Matthew-Collins-Defra) [inline node/risk-app/server/plugins/views.js:54]: All reading of environment variables should be done in the config module. I'm unsure why this change to check `process.env` is here, but this is the wrong place for it. This all needs to be done by the config module so that it can be mocked in tests, and there is a single place for reading validating config values.

File: node/risk-app/server/src/js/map-page/map-controls.js [hunk 2]
```diff
@@ -73,15 +76,15 @@ export function toggleAdvancedOptions () {
     mapControlsConsts.reservoirsContainer.classList.add(mapControlsConsts.keyAdvOptionsContainer)
     mapControlsConsts.reservoirsContainer.classList.remove(mapControlsConsts.keyContainer)
   } else {
-    if (window.location.href.includes('map=SurfaceWater')) {
+    if (currentMapPage === 'map=SurfaceWater') {
```
Comment (Matthew-Collins-Defra) [inline node/risk-app/server/src/js/map-page/map-controls.js:79]: Is this correct? I thought that UrlSearchParams would return just the value of the parameter, not the whole string.

File: node/risk-app/server/src/js/map-page/map-page.js [hunk 3]
```diff
@@ -192,12 +192,15 @@ handleArrowClick(mapPageConsts.leftArrow, mapPageConsts.leftMove)
 handleScroll(mapPageConsts.scenarioSelectionDepth, [mapPageConsts.rightArrow[0], mapPageConsts.leftArrow[0]])
 handleScroll(mapPageConsts.scenarioSelectionVelocity, [mapPageConsts.rightArrow[1], mapPageConsts.leftArrow[1]])
 
+const searchParams = new URLSearchParams(window.location.search)
+const currentMapPage = searchParams.get('map')
+
 function getInitialKeyOptions () {
-  if (window.location.href.includes('map=SurfaceWater')) {
+  if (currentMapPage === 'SurfaceWater') {
```
Comment (Matthew-Collins-Defra) [inline node/risk-app/server/src/js/map-page/map-page.js:199]: This is not consistant with the use of this Class in map-controls.js

File: node/risk-app/server/src/js/map-page/screen-size-adjustments.js [hunk 4]
```diff
@@ -22,14 +22,17 @@ function adjustLogoAndCopyright () {
...
+
 function keyToggleAdjustments () {
   if (!screenAdjustConsts.keyDisplay.classList.contains('hide') && window.innerWidth <= screenAdjustConsts.deviceScreenWidth) {
     screenAdjustConsts.scenarioSelectionDepth.classList.add('hide')
     screenAdjustConsts.scenarioSelectionVelocity.classList.add('hide')
   } else if (!screenAdjustConsts.keyDisplay.classList.contains('hide') && window.innerWidth > screenAdjustConsts.deviceScreenWidth) {
-    if (window.location.href.includes('map=RiversOrSea') ||
-    window.location.href.includes('map=SurfaceWater') ||
-    window.location.href.includes('map=Reservoirs')) {
+    if (currentMapPage === 'map=RiversOrSea' ||
```
Comment (Matthew-Collins-Defra) [inline node/risk-app/server/src/js/map-page/screen-size-adjustments.js:33]: See previous comments about UrlSearchParams

**Review by Matthew-Collins-Defra — CHANGES_REQUESTED** [review]

File: node/risk-app/server/src/js/map-page/map-controls.js (hunk 2 above)
Comment (MickStein) [reply node/risk-app/server/src/js/map-page/map-controls.js:79]: It does but there are 3 params and we only need to check which map we have entered.

**Review by Matthew-Collins-Defra — APPROVED** [review]
lgtm
